src/org/traccar/reports/ReportUtils.java

```diff
@@ -35,4 +39,46 @@ private ReportUtils() {
         return result;
     }
 
+    public static double calculateDistance(Position firstPosition, Position lastPosition) {
+        return calculateDistance(firstPosition, lastPosition, true);
+    }
+
+    public static double calculateDistance(Position firstPosition, Position lastPosition, boolean useOdometer) {
+        BigDecimal distance = new BigDecimal("0.0");
```

Comment:
We should not round anything on the server side. All the formatting should be on the client, so I guess we can use double here.